test/unit3/test_strptime.py

```diff
@@ -83,8 +83,8 @@ def test_date_time(self):
         #  output.
         magic_date = (1999, 3, 17, 22, 44, 55, 2, 76, 0)
         strftime_output = time.strftime("%c", magic_date)
-        self.assertEqual(time.strftime(self.LT_ins.LC_date_time, magic_date),
```

Comment:
And time.strftime(self.LT_ins.LC_date_time, magic_date) is set to:
`Wed 17 Mar 1999 10:44:55 PM australian eastern daylight time`

hence the test fails

Reply:
These tests seem to be failing only because of the case, not because of any issues with daylight savings.